### ISSUE TITLE
General plan outline

### ISSUE THREAD

**Issue by miguelgfierro** [Azure/fast_retraining#32]
Contents of MVP2:
- [x] Optimize airline experiment
- [x] Add comparison with XGBoost hist
- [x] Add Guolin experiments and run them on the same machine
- [x] Update notebooks with new parameters
- [x] Add experiments with GPU
- [x] Change notebooks with bokeh
- [x] Blog writeup

**Comment by miguelgfierro** [comment]
TODO cleanup:
-airline: remove evaluation
-football remove evaluation
-remove xgboost hist and leave the loader
-change `df_plane_numeric_small = df_plane_numeric.sample(n=10000000).reset_index(drop=True)` to `df_plane_numeric_small = df_plane_numeric.sample(n=1e6).reset_index(drop=True)`


-remember gotchas: 
1) evaluation will slow the training
2) custom loss functions in python are also slower
3) ligthgbm is cache aware

**Comment by miguelgfierro** [comment]
TODO GPU:
- finish parameters in both airline and higgs
- code results for different dataset sizes
- add GPU times and perf to the LightgbmvsXgboost notebook 
- also the results for different dataset sizes

TODO bokeh:
- save as png and render it to github
- change requirements with recent version of bokeh

TODO blog:
- export bokeh plots to htlm+js 
-try to render bokeh in the post blog 
-think about the normal post
-think about Q&A
- decide if put Q&A together or in a different post
